src/selectedtests/app/helpers.py

```diff
@@ -0,0 +1,31 @@
+"""Helper functions for selected tests API."""
```

Comment:
[nit] I'm not a fan of files named "helpers". It would be nice if this was a more descriptive name or these functions were in more descriptive places.